README.md

```diff
@@ -1,2 +1,4 @@
 # ecommerce-python-sql
 This repo is used for practicing FastAPI + MySQL
```

Comment:
Add trailing spaces to end the line. Refer to [Markdown Syntax](https://www.markdownguide.org/getting-started/)

Reply:
Updated